runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsDestinationOverlay.java

```diff
@@ -35,24 +35,32 @@
 import net.runelite.client.ui.overlay.OverlayPosition;
 import net.runelite.client.ui.overlay.OverlayPriority;
 import net.runelite.client.ui.overlay.OverlayUtil;
+import javax.inject.Inject;
 
-public class TileIndicatorsOverlay extends Overlay
+public class TileIndicatorsDestinationOverlay extends Overlay
 {
 	private final Client client;
 	private final TileIndicatorsConfig config;
 
-	TileIndicatorsOverlay(Client client, TileIndicatorsConfig config)
+	@Inject
+	private TileIndicatorsDestinationOverlay(Client client, TileIndicatorsConfig config)
 	{
-		this.client = client;
-		this.config = config;
 		setPosition(OverlayPosition.DYNAMIC);
 		setLayer(OverlayLayer.ABOVE_SCENE);
 		setPriority(OverlayPriority.LOW);
+
+		this.client = client;
+		this.config = config;
```

Comment:
Don't reformat code for no reason. Also, the variables in constructor should be assigned before any custom properties, so do that. And do same in new overlay too